code/Items/Data/ItemClothes.cs

```diff
@@ -13,10 +13,8 @@ public ItemClothes(string id, string name, string model, ClothesSlot slot) : bas
 
         private ClothesSlot slot;
 
-        public override int MaxStack => 1;
-        public override bool DestroyOnUse => true;
 
-        public override void UsedBy(MainPlayer player)
+        public override void UsedBy(MainPlayer player, ItemStack itemStack)
```

Comment:
Ya plus d'item stack normalement

Reply:
When we write the behaviour of an Item, we should know the origin of the action.
It's necessary to get the custom data of the ItemStack